data-sources/botolytics/config.yml

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Since we're deciding to go with guided install data source we can also delete this.